### ISSUE TITLE
When lava is used as fuel, the fuel indicator does not move until the 31st item is smelted

### ISSUE THREAD

**Issue by ghost** [LO6AN/New-Frontier-Craft#2]
in the furnace when you use lava as a source only once it gets to the 31th smelted item you will see the fuel icon disposing
![2021-03-07_10 35 34](https://user-images.githubusercontent.com/67587410/110235564-03def300-7f31-11eb-8853-5951774a3570.png)

**Comment by Roadhog360** [comment]
I have no idea what this means

**Comment by ghost** [comment]
> I have no idea what this means

the fuel indicator will not move until it reaches its 31th item that it's smelting

**Comment by Roadhog360** [comment]
So it turns out this happens due to the fuel times on a brick oven being longer. Will adjust the fuel times accordingly.

**Comment by LO6AN** [comment]
> 
> 
> So it turns out this happens due to the fuel times on a brick oven being longer. Will adjust the fuel times accordingly.

What do you mean by "Brick Oven". The bug report is in the context of the furnace, not the oven. What's the one got to do with the other, they use completely different sets of code for their burning mechanics.

**Comment by VivianTheFox** [comment]
Not sure what Road is saying about the Brick Oven, as this issue is related to the furnace. I can't replicate this bug on my end, so I'm gonna throw a low priority and can't reproduce tag on this.
